### DISCUSSION TITLE
Blog page needs a "home" page

### DISCUSSION THREAD

**Discussion by PhilPNguy** [uptane/uptane.github.io#152] [Ideas]
Right now, https://uptane.org/blog has all posts in one long scroll. This is perfectly fine for now since we don't have much, but eventually we can run into scaling issues with a lot of posts.

I suggest we have a dedicated "Home" page to quick access recent blogs or "see all"
Alternatively we could keep this scroll but have pagination and limit to the past 20 (or any number) blog posts.

This does depend on how docusaurus handles the pagination or if it has any other interesting ways to do it instead.

**Comment by hexsecs** [comment]
@Abhijay007 created a pull request for this #169
